filecoin-proofs/src/api.rs

```diff
@@ -759,6 +774,61 @@ mod tests {
         Ok(())
     }
 
+    #[test]
+    fn test_verify_seal_fr32_validation() {
+        //        verify_seal() // comm_r, comm_d, comm_r_star
```

Comment:
why is this commented outd?

Reply:
@sidke It should be deleted, actually.

Reply:
...or rewritten to actually test `verify_seal` instead of `verify_post`. :)